packages/datatrak-web/src/utils/toast.ts

```diff
@@ -18,3 +18,14 @@ export const errorToast = (message: string) => {
     hideIconVariant: true,
   });
 };
+
+type Options = OptionsObject & { Icon?: OptionsObject['Icon']; hideCloseButton?: boolean };
+
+export const infoToast = (message: string, options?: Options) => {
+  enqueueSnackbar(message, {
```

Comment:
Seems like naming got out of sync somewhere. Presuming _toast_ and _snackbar_ refers to the same thing, should these names be consolidated?

Reply:
The discrepancy is because we use the term toast and the underlying library we use calls the component snackbar. This toast file does consolidate the name though as it's only through these utils that we call the library.